core/templates/dev/head/pages/exploration_editor/translation_tab/state_translation_directive.html

```diff
@@ -1,17 +1,33 @@
 <div class="oppia-page-card oppia-state-translation-page">
-  <div class="tarnslation-nav md-tab">
+  <div class="translation-nav md-tab">
     <ul class="oppia-translation-tabs">
       <li ng-class="{'oppia-active-translation-tab': isActive(TAB_ID_CONTENT)}">
-        <a ng-click="onTabClick(TAB_ID_CONTENT)" href="" class="oppia-translation-tabs-text">Content</a>
+        <a ng-click="onTabClick(TAB_ID_CONTENT)" href="" ng-style="tabStatusColorStyle(TAB_ID_CONTENT)" class="oppia-translation-tabs-text">
+          <span ng-show="tabNeedUpdatesStatus(TAB_ID_CONTENT)" style="color: red"
+                uib-tooltip="Audio needs update!" tooltip-placement="top">⚠
```

Comment:
Can you include a call to action? Otherwise it could still be unclear. So maybe something like: "Audio needs update to match text. Please record new audio."

Reply:
Done!